src/stories/initialData.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+// eslint-disable-next-line max-len
+export const INITIAL_DATA = `<h2>CKEditor5</h2><hr><h3>Basic styles</h3><p>Text can have <strong>bold</strong>, <i>italic</i>, and <u>underline</u> content. <code>Inline code</code> is also available. <a href="https://www.google.com">Links work as well.</a></p><ul><li>Lists</li><li>Are<ol><li>Nestable</li><li>And can change</li><li>Types</li></ol></li></ul><h3>Block elements</h3><blockquote><p>Block quotes can have inline content</p><p><strong>- Author name</strong></p></blockquote><pre><code class="language-plaintext">Preformatted code can go here</code></pre>`;
```

Comment:
Please use "CKEditor" throughout. Primarily for maintenance, as someday the version will change. (Secondarily, CKSource writes it as "CKEditor<space>5".)